rpm/caddy.spec

```diff
@@ -0,0 +1,134 @@
+# https://bugzilla.redhat.com/show_bug.cgi?id=995136#c12
+%global _dwz_low_mem_die_limit 0
+
+Name:           caddy
+# https://docs.fedoraproject.org/en-US/packaging-guidelines/Versioning/#_versioning_prereleases_with_tilde
+Version:        2.0.0~beta6
+%global tag     v2.0.0-beta6
+Release:        1%{?dist}
+Summary:        HTTP/2 web server with automatic HTTPS
```

Comment:
FWIW, in the future I think I want to make summaries/descriptions of the project more generic, something like "Web server with automatic HTTPS" (since versions will get outdated)

Will still merge of course, just wanted to mention that

Reply:
Sure, that's an easy adjustment.